src/index.html

```diff
@@ -138,7 +146,35 @@ <h2 class="modal__title">New post</h2>
         <p class="modal__caption"></p>
       </div>
     </div>
-    <script defer src="./scripts/index.js"></script>
-    <script defer src="./scripts/validation.js"></script>
+    <div class="modal" id="avatar-modal">
+      <div class="modal__container">
+        <button type="button" class="modal__close-btn"></button>
+        <h2 class="modal__title">Edit avatar</h2>
+        <form name="edit-avatar" class="modal__form" novalidate>
+          <label for="profile-avatar-input" class="modal__label">
+            Avatar
+            <input
```

Comment:
* Please, add `type="url"` for the `avatar` input, to validate it as a `link` https://snipboard.io/9ZTsdn.jpg 

```html

            <input
              type="url"
```